Standards/scs-0112-v1-volume-type-standard.md

```diff
@@ -0,0 +1,220 @@
+
+Quality of Service parameters for volumes can be defined in a volume qos object. While admins can use these qos objects directly on volumes, a user is not able to see such an object. Instead there can be ONE volume qos object associated to a volume type, which is then be used on all volumes created from that volume type. Through this indirection, volume types also working without these associated qos objects and this feature not being heavily used, the SCS standard will currently only view this as an OPTIONAL volume type feature.
+
+To make users aware that a volume type includes specific qos options, we recommend to write it into the description of a volume type, as any association to a volume qos object cannot be seen by normal users:
+
+```text
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field              | Value                                                                                                                                                        |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| access_project_ids | None                                                                                                                                                         |
+| description        | Content will be replicated three times on spinning disks to ensure consistency and availability for your data. Iops: read:20k, write:10k               |
+| id                 | d63307fb-167a-4aa0-9066-66595ea9fb21                                                                                                                         |
+| is_public          | True                                                                                                                                                         |
+| name               | hdd-three-replicas-AZ134                                                                                                                                     |
+| properties         |
+                                  |
+| qos_specs_id       | None                                                                                                                                                         |
++--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+## Related Documents
+
+[Here is the decision record document.](https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0111-v1-volume-type-decisions.md)
+
+## Conformance Tests
+
+As there are currently no REQUIRED volume types, we can only look for the RECOMMENDED aspects and thus executing a conformance test is not a must.
```

Comment:
Now I'm confused. In line 44 you wrote, that there is always a default volume type. Doesn't this mean, that there is a required volume type?

Reply:
no. The default volume type is necessary through openstack. Like a server that always has a flavor, a volume always needs a volume type. But to be backward compatible the volume type must not needed to be specified. That requires the presence of a "default" volume type in openstack, but not in scs.